### PR TITLE
OOP school library: preserve data

### PR DESCRIPTION
**Pull request by samiullahbahadur**
### **Project requirements**
**_we did the below tasks:_**

- [ ] Check the documentation for Ruby's [JSON library](https://ruby-doc.org/stdlib-3.0.0/libdoc/json/rdoc/JSON.html).
- [ ] On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)
- [ ] On startup of your app load all data from files. Remember to handle the case when one or more of the files are missing.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,165 +1,56 @@
+require './books'
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './people'
+require './rentingbooks'
```
Comment (Lameck1) [inline app.rb:8]: - [ ] Everything is working fine but this line gives this error 👇🏼:

  image
  
  This is just a typo issue, the file you are trying to import is named as `retingbooks` but you are importing it as `rentingbooks`.  The name of the file has the letter `n` in the word `renting`. Please look into this and make the proper refactoring.

**Review by Lameck1 — APPROVED** [review]
Hi, @samiullahbahadur & @Mutalenic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (samiullahbahadur) [reply app.rb:8]: Okay, I did fix

**Review by elyor-doniyorov — APPROVED** [review]
Hi @samiullahbahadur and @Mutalenic, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**